packages/manager/modules/pci/src/projects/project/quota/quota.html

```diff
@@ -248,5 +248,22 @@
             >
             </span>
         </oui-datagrid-column>
+        <oui-datagrid-column data-title="$ctrl.PRODUCTS.FLOATING_IP">
+            <span data-ng-bind="$row.network.usedFloatingIPs"></span>
+            <span class="mx-1">/</span>
```

Comment:
**question**: a simple space isn't enough ?

Reply:
there's a `/` char as well not just a space.

Reply:
Yep I've noticed it, but what I meant was html native spaces isn't enough around this slash instead of mx-1 class ?
```suggestion
           /
```

Reply:
Just a native space in between 2 spans can make it less readable